browser/index.js

```diff
@@ -118,7 +116,9 @@ SocketPeer.prototype.connect = function () {
   self.socket.onopen = function () {
     self.pair();
   };
-  self.socket.onerror = self._socketError.bind(self);
+  self.socket.onerror = function (event) {
```

Comment:
no, `onerror` emits an event not an `Error` object. `event.data` is the `Error` object (if there even is one [P.S. there's not always one]).